docs/roman/ramp_fitting/description.rst

```diff
@@ -0,0 +1,231 @@
++-------------------+------------------------+----------+
+| 20                | 50                     | 3        |
++-------------------+------------------------+----------+
+| 50                | 100                    | 6        |
++-------------------+------------------------+----------+
+| 100               |                        | 10       |
++-------------------+------------------------+----------+
+
+Segment-specific Computations:
+------------------------------
+The variance of the slope of a segment due to read noise is:
+
+.. math::  
+   var^R_{s} = \frac{12 \ R^2 }{ (ngroups_{s}^3 - ngroups_{s})(tgroup^2) } \,,
+
+where :math:`R` is the noise in the difference between 2 frames, 
+:math:`ngroups_{s}` is the number of groups in the segment, and :math:`tgroup` is the group 
+time in seconds (from the keyword TGROUP).  
+
+The variance of the slope in a segment due to Poisson noise is: 
+
+.. math::  
+   var^P_{s} = \frac{ slope_{est} }{  tgroup \times gain\ (ngroups_{s} -1)}  \,,
+
+where :math:`gain` is the gain for the pixel (from the GAIN reference file),
+in e/DN. The :math:`slope_{est}` is an overall estimated slope of the pixel,
+calculated by taking the median of the first differences of the groups that are
+unaffected by saturation and cosmic rays. This is a more
+robust estimate of the slope than the segment-specific slope, which may be noisy
+for short segments. 
+
+The combined variance of the slope of a segment is the sum of the variances: 
+
+.. math::  
+   var^C_{s} = var^R_{s} + var^P_{s}
+
+
+Exposure-level computations:
+----------------------------
+
+The variance of the slope due to read noise is: 
+
+.. math::  
+   var^R_{o} = \frac{1}{ \sum_{s} \frac{1}{ var^R_{s}}} 
+
+where the sum is over all segments.
+
+
+The variance of the slope due to Poisson noise is: 
+
+.. math::  
+   var^P_{o} = \frac{1}{ \sum_{s} \frac{1}{ var^P_{s}}}
+
+The combined variance of the slope is the sum of the variances: 
+
+.. math::  
+   var^C_{o} = var^R_{o} + var^P_{o}
+
+The square root of the combined variance is stored in the ERR array of the primary output.
+
+The overall slope depends on the slope and the combined variance of the slope of all
+segments, so is a sum over segments:
+
+.. math::    
+    slope_{o} = \frac{ \sum_{s}{ \frac{slope_{s}} {var^C_{s}}}} { \sum_{s}{ \frac{1} {var^C_{s}}}}
+
+
+Upon successful completion of this step, the status keyword S_RAMP will be set
+to "COMPLETE".
+
+
+Error Propagation
+=================
+
+Error propagation in the ramp fitting step is implemented by storing the
+square-root of the exposure-level combined variance in the ERR array of the primary
+output product. This combined variance of the exposure-level slope is the sum
+of the variance of the slope due to the Poisson noise and the variance of the 
+slope due to the read noise. These two variances are also separately written
+to the extensions VAR_POISSON and VAR_RNOISE in the primary output.
```

Comment:
Should extensions be arrays, 
... written to the arrays VAR_POISSON and VAR_RNOISE in the asdf file.

Reply:
Yes, "arrays"